website/docs/style_guide/about_style_guide.md

```diff
@@ -0,0 +1,65 @@
+# Robot Framework Style Guide
+
+## Purpose
+
+This style guide is a community driven set of sensible rules to write your Robot Framework code.
+
+As a starting point the existing standards from [Robot Framework](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#getting-started) user guide, [Robocop](https://github.com/MarketSquare/robotframework-robocop) and [Robotidy](https://robotidy.readthedocs.io/en/stable/).
+
+## Robocon 2022 Presentation
+
+[Robot Framework is not ...](https://youtu.be/Mpt_4MItha0)
+
+## Contributors
+
+Guido Demmenie, Mannana Koberidze, Kelby Stine
+
+### Special Thanks To
+
+Václav Fuksa, Many Kasiriha, Bartlomiej Hirz, Mateusz Nojek, René Rohner, Miikka Solmela
+
+### We are always looking for contributors
+
+- We meet up once every two weeks to talk about style guide topics
+- Reach out to any one of us on the Slack channels
+
+---
+
+## Basic Rules and Guidelines
+
+Here are some good basic guidelines for formatting your Robot Framework codes. As these are guidelines, use your best judgment in cases not covered by these recommendations.
+
+- Separate pieces of the data, such as keywords and their arguments, with four (4) spaces.
+
+- Robot and Resource file section order:
+
+```robot
+*** Comments ***
+
+*** Settings ***
+
+*** Variables ***
+
+*** Test Cases / Tasks ***
+(not available in resource files)
+
+*** Keywords ***
+```
+
+- Keywords within resource files should be alphabetized.
+- The recommended minimum line length is 120.
```

Comment:
Not maximum?

Reply:
Yes, this should be maximum.

Reply:
It should actually be the minimum suggested maximum line length.